test/command/suite/check/pat/default.test

```diff
@@ -0,0 +1,6 @@
+table_create --name Users --flags TABLE_PAT_KEY --key_type ShortText
+load --table Users
+[
+{"_key":"Alice"}
+]
+check Users
```

Comment:
```suggestion
check --obj Users
```

I think the http test will be successful with this change.
(The `.expected` needs to be updated as well.)